### DISCUSSION TITLE
Review of Speech

### DISCUSSION THREAD

**Discussion by AlexeyMatveev686** [ONLYOFFICE/onlyoffice.github.io#226] [Polls]
<img src="https://raw.githubusercontent.com/ONLYOFFICE/onlyoffice.github.io/master/discussions-icons/speech.svg">

**Comment by na-sowas** [comment]
Doesn't work in Desktop version (at least not when I tried)

**Comment by stone-alex** [comment]
Requires cloud. does now work with local TTS like Piper-TTS

**Comment by Misokados** [comment]
Hello @stone-alex @na-sowas 
I have checked the Desktop app v.9.2.1 and the Speech v.2.0.2. All things are OK. Please describe the issue as detailed as possible.
To avoid confusion in this thread, you can create a separate post on https://community.onlyoffice.com/ and describe the issue you faced.

**Comment by stone-alex** [comment]
See screen shot attached.


On Mon, 2026-02-09 at 02:20 -0800, Misokados wrote:
> Speech v.2.0.2

-- 
_________________________
Alex Stone

**Reply by Misokados** [reply]
Please attach a screenshot.
